### PR TITLE
feat: add inspectManifest endpoint to api

### PR DESCRIPTION
**Pull request by cdrage**
feat: add inspectManifest endpoint to api

### What does this PR do?

* Adds the inspectManifest endpoint to the API
* Requires https://github.com/containers/podman-desktop/pull/6812 to go in
first

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop-extension-bootc/issues/340

### How to test this PR?

Use the `inspectManifest` endpoint with a passed in image.

### CODE REVIEW

**Comment by cdrage** [comment]
Do not merge until the following are done:

- [ ] https://github.com/containers/podman-desktop-extension-bootc/pull/264 merged
- [ ] https://github.com/containers/podman-desktop/pull/6812 merged
- [ ] Podman Desktop API updated in bootc

**Comment by cdrage** [comment]
Works now! Ready for review 💯 

FYI we will need to update the Podman Desktop dependency to 1.10 for this to work, I've opened up issue https://github.com/containers/podman-desktop-extension-bootc/issues/352 to make sure we complete it.

File: packages/frontend/src/Build.spec.ts [hunk 1]
```diff
@@ -60,6 +60,7 @@ const mockBootcImages: ImageInfo[] = [
     Size: 0,
     Containers: 0,
     SharedSize: 0,
+    Digest: 'sha256:image1',
```
Comment (deboer-tim) [inline packages/frontend/src/Build.spec.ts:63]: Pinged Charlie to confirm - updating PD API causes Digest to be a required field, hence the need to add it here and other places.

File: packages/shared/src/BootcAPI.ts [hunk 2]
```diff
@@ -17,14 +17,15 @@
...
-import type { ImageInfo, ImageInspectInfo } from '@podman-desktop/api';
+import type { ImageInfo, ImageInspectInfo, ManifestInspectInfo } from '@podman-desktop/api';
 
 export abstract class BootcApi {
   abstract checkPrereqs(): Promise<string | undefined>;
   abstract buildExists(folder: string, types: BuildType[]): Promise<boolean>;
   abstract buildImage(build: BootcBuildInfo, overwrite?: boolean): Promise<void>;
   abstract pullImage(image: string): Promise<void>;
   abstract inspectImage(image: ImageInfo): Promise<ImageInspectInfo>;
+  abstract inspectManifest(image: ImageInfo): Promise<ManifestInspectInfo>;
```
Comment (deboer-tim) [inline packages/shared/src/BootcAPI.ts:28]: Discussed with Charlie - I don't like this (vs having engineId, id parameters), but it is convenient for UI and consistent with other functions here. API is not final, so fine for now at least.

**Review by deboer-tim — APPROVED** [review]

**Pull request merged**